src/uct/api/addr.h

```diff
@@ -32,7 +32,7 @@ enum {
 typedef struct uct_sockaddr_process {
     UCT_SOCKADDR_COMMON (sp_);
     uint64_t   node_guid;
-    uint64_t   cookie;
+    uint64_t   id;
```

Comment:
@shamisp yes - i think we need to make sure API changes have a working implementation before PR-ing them.